### ISSUE TITLE
O índice está desatualizado

### ISSUE THREAD

**Issue by fabiopapais** [ifpeopensource/ifpeopensource.github.io#1]
Novos projetos foram adicionados, portanto o índice precisa ser atualizado

**Comment by Zepelino** [comment]
Eu quero ajudar, Onde está o "índice"?

**Comment by fabiopapais** [comment]
O índice é próprio REAME.md com a lista de projetos do IFPE Open Source. Basta adicionar os projetos restantes seguindo o formato atual.

**Comment by Zepelino** [comment]
No caso dos projetos que não tenham um site, eu coloco o link do repositório?

**Comment by fabiopapais** [comment]
Exatamente, se também puder colocar a logo como nos outros exemplos seria ótimo :)
